### DISCUSSION TITLE
Use absolute time in GitHub Actions workflow display

### DISCUSSION THREAD

**Discussion by scott-doyland-burrows** [community/community#14399] [Actions]
Hi,

I see this in GHA:

![image](https://user-images.githubusercontent.com/61512176/161997652-861003ee-aa5a-4be1-81ff-4a06d5a6b22b.png)

Note the time on the right is relative, it shows the runs as 2 days ago.

Could it also show the absolute start time (and indicate if it is UTC etc), or maybe an option to either show relative or absolute.

Most times I have lots of workflow runs per day, and seeing 2 days ago for them all is not that helpful.

Thanks.

**Comment by ethomson** [comment]
Thanks, @scott-doyland-burrows - I've also found that I'd like to be able to get the exact time from this page. I'll talk to the team and see if there's a way that we could surface some more information, and we'll put this on the backlog.

**Comment by 7thFox** [comment]
Seconding this. I shouldn't have to do math to figure out when my build went in. It's not even listed if I open the action, still just tells me "17 hours ago"

**Comment by NedWilbur** [comment]
+1, trying to cross reference times with other events is a pain as it is now.

**Comment by jan-konecny-deltatre** [comment]
Is there still no progress on this? Did not find any update or setting for this yet. Would be really helpful.

**Comment by tyleragnew** [comment]
Would really like this! Trying to debug an issue now but having to sift through many deploys from "2 weeks ago"
